src/components/artwork/details.tsx

```diff
@@ -19,7 +19,7 @@ export class ArtworkDetails extends React.Component<DetailsProps, null> {
     } else if (artists && artists.length) {
       let artistsEl = []
       for (let i = 0; i < artists.length; i++) {
-        artistsEl.push(<TextLink href={ artists[i].href }>{ artists[i].name }</TextLink>)
+        artistsEl.push(<TextLink href={ artists[i].href } key={ i }>{ artists[i].name }</TextLink>)
```

Comment:
A simple index should not be used as the key, because we’d quickly run into duplicate keys on a full page and React keeps track (and re-uses) elements by this key.

In general I advice to use an object’s `__id` key, because it’s globally unique across all models _and_ Relay already automatically fetches this field even if you don’t explicitly include it in your query (thus adding it to the query does not add more data that’s to be fetched).